core/arch/arm/mm/mobj.c

```diff
@@ -606,6 +606,7 @@ static TEE_Result mobj_with_fobj_get_pa(struct mobj *mobj, size_t offs,
 
 	return TEE_SUCCESS;
 }
+KEEP_PAGER(mobj_with_fobj_get_pa);
```

Comment:
Doesn't this pull in `mobj_with_fobj_ops` and all the other functions referenced via `to_mobj_with_fobj()`?
It will only happen with asserts and given that we don't seem to care at other places in this file, maybe it's not a problem.

Reply:
Attribute `__rodata_unpaged` does not generate a dependency that bring all ops functions in unpaged sections. And that's fine, actually, the ops structure must be unpaged because of `mobj_is_paged()` below, but the ops function do not need to be unpaged... but `mobj_with_fobj_get_pa()` here.

I can reproduce the issue on `qemu_virt`: OP-TEE core crash during `xest 1010` when I build with:
```shell
make CFG_WITH_PAGER=y CFG_WITH_LPAE=y \
     CFG_TEE_CORE_DEBUG=n CFG_UNWIND=y CFG_TEE_CORE_LOG_LEVEL=2 \
     CFG_CORE_TZSRAM_EMUL_SIZE=294912 run
```
I admit it's a strange config. I also easily reproduce the issue on my stm32mp1 which SRAM is only 256kB.